Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -2,94 +2,166 @@
 {
     public static class ArrayOperations
     {
-        /// <summary>
-        /// Sort the array in ascending order.
-        /// If array empty or null- don't do anything.
-        /// </summary>
-        /// <param name="array">Input array in a random order.</param>
         public static void Sort(int[] array)
         {
-            // ToDo: implement.
+            if (array != null && array.Length != 0)
```

Comment:
Try inverting the if statements and return. This reduces the nested scopes number.
```
if(array == null || array.Legnth == 0) return;
int t; //...
```